02-write-your-first-program/questions/01-gopath/README-edit.md

```diff
@@ -0,0 +1,57 @@
+# Questions: Run Your First Program
```

Comment:
Fixed:
---
> Wrong file used here.. swapping for the correct `01-gopath/README-edit.md` file.
---
# Questions: Go PATH #

---

## 1. Where should you save your Go source code files? ##

(A) Anywhere on my computer

(B) In the `$GOPATH` directory

(C) In the `$GOPATH`/src directory

---

**Solution**: (C)

---

## 2. What is the `$GOPATH`? ##

(A) It's the file Go uses to run your programs.

(B) It is the place where Go stores all source code files and compiled packages.

(C) It's a path for all gophers to follow.

---

**Solution**: (B)

---

## 3. How can you print your `$GOPATH` in the terminal? ##

(A) Using the `ls` command.

(B) Using the `go env GOPATH` command

(C) Using the `go environment` command

---

**Solution**: (B)

That's right! By running the command `go env GOPATH` your Go PATH variable should print to the console screen.

To view all of your Go environment variables, run the command `go env`.

---